webapp/packages/plugin-data-spreadsheet-new/src/DataGrid/DataGridTable.tsx

```diff
@@ -452,6 +456,21 @@ export const DataGridTable = observer<IDataPresentationProps>(function DataGridT
     }
   };
 
+  function handleSort(attributePosition: number, multiple?: boolean) {
+    const currentOrder = constraintsAction!.getOrder(attributePosition);
```

Comment:
This will not work as expected because `attributePosition` is a position of the column in the grid (in the grid, we have one more "virtual" column for row index)
Please check `isCellEditable`, for example, to understand how to get the column key that can be used in `getColumnInfo` that can be used to get "position"